internal/backend/user.go

```diff
@@ -107,6 +109,8 @@ func (user *user) deleteAllMessagesMarkedDeleted(ctx context.Context) error {
 			return err
 		}
 
-		return user.store.Delete(ids...)
+		return user.store.Delete(xslices.Map(ids, func(i imap.InternalMessageID) string {
```

Comment:
i think `store` should have a strongly typed interface too, since it operates on internal message IDs, do you agree?

Reply:
:+1:

Reply:
Done